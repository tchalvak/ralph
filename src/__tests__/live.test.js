import React from 'react'
import renderer from 'react-test-renderer'
import Live from '../pages/live'
import { StaticQuery } from 'gatsby'

beforeEach(() => {
    StaticQuery.mockImplementationOnce(({ render }) =>
        render({
            site: {
                siteMetadata: {
                    title: `Ralph AI`,
                },
            },
        })
    )
})

describe('Live', () => {
    it('renders correctly', () => {
        const location = {
            pathname: '/',
        }

        const tree = renderer.create(<Live location={location} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
