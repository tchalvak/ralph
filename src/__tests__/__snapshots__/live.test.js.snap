// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Live renders correctly 1`] = `
<div
  className="page"
  id="wrapper"
>
  <div>
    <a
      href=""
    >
      <button
        type="button"
      >
        View the Presentation Live
      </button>
    </a>
  </div>
</div>
`;
